data/volunteers.js

```diff
@@ -9,11 +9,36 @@ export default [
     title: 'Front-End Developer',
     image: 'simi-damani.jpg',
   },
+  {
+    name: 'Nick Holden',
+    title: 'Software Engineer',
+    image: 'nick-holden.jpg',
+  },
+  {
+    name: 'Dashiel Lopez Mendez',
+    title: 'Software Engineer',
+    image: 'dashiel-lopez-mendez.jpg',
+  },
+  {
+    name: 'Daniel Olivares',
+    title: 'Senior Software Engineer',
+    image: 'daniel-olivares.jpg',
+  },
   {
     name: 'Jonathan Pascoe',
     title: 'Geographic Information Systems Professional',
     image: 'jonathan-pascoe.jpg',
   },
+  {
+    name: 'Athula Pudhiyidath',
+    title: 'Graduate Student Researcher',
```

Comment:
There was no title in the image name in the Google Drive, so I Googled Athula and this seemed like an appropriate title. Would you prefer something else, @athpud?

Reply:
Hi @nholden, sorry about the missing title -- please use "Data Scientist" for @athpud -- thanks!